packages/slick-components/src/Utils.js

```diff
@@ -0,0 +1,33 @@
+function getSlideListWidth(slick) {
```

Comment:
I prefer not use the name `Utils` for this file. Something link `ItemsPerPage` or `PageItems` will be better.